src/textual/css/stylesheet.py

```diff
@@ -316,7 +316,7 @@ def replace_rules(
         styles = node.styles
         base_styles = styles.base
 
-        # Styles currently used an new rules
+        # Styles currently used on new rules
```

Comment:
quick typo fix as I was reading code here and there to try to understand how to add a new CSS property :-)